### PR TITLE
symmetry_axis parameter for `get/put_image_quadrant()` functions

### PR DESCRIPTION
**Pull request by stggh**
#103 point 3:
 In `abel.tools.symmetry.py` for the two functions `abel.tools.symmetry.get_image_quadrants()`, `abel.tools.symmetry.put_image_quadrants()`:

 replace `vertical_symmetry` and `horizontal_symmetry` parameters with a `symmetry_axis` parameter as per #88 @DanHickstein's code, near bottom.    

Updated `abel.transform()` and examples.

Edit: quadrant combination is now an average as per #88.

Added bonus:  I added Classes to `example/example_simple_GUI.py`, which made it not so simple, and so named it `examples/example_GUI.py` and deleted `example/example_simple_GUI2.py`

Note, unable to test docs

### CODE REVIEW

File: abel/tools/symmetry.py [hunk 1]
```diff
@@ -21,79 +20,103 @@ def get_image_quadrants(IM, reorient=False, vertical_symmetry=False,
...
+        |   *    |    *   |
+        | Q2  *  | *   Q3 |          cQi == averaged combined quadrants
+        +--------+--------+                 
+
+
+       symmetry_axis = None - individual quadrants
+
+       symmetry_axis = 0 (vertical) - average Q0+Q1, and Q2+Q3
+
+       symmetry_axis = 1 (horizonat) - average Q1+Q2, and Q0+Q3
```
Comment (DanHickstein) [inline abel/tools/symmetry.py]: typo here

File: doc/abel.rst [hunk 2]
```diff
@@ -78,24 +78,6 @@ abel.tools.center module
     :undoc-members:
     :show-inheritance:
 
-abel.tools.io module
```
Comment (DanHickstein) [inline doc/abel.rst]: Shouldn't these be included in the docs?

File: doc/conf.py [hunk 3]
```diff
@@ -149,7 +151,7 @@ def __getattr__(cls, name):
 #html_theme_options = {}
 
 # Add any paths that contain custom themes here, relative to this directory.
-#html_theme_path = []
+html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
```
Comment (DanHickstein) [inline doc/conf.py]: What does this do?

**Comment by DanHickstein** [comment]
Thanks for doing this @stggh!

I think that you will also need to update two of the test functions to get this to pass the unit tests. Use 
```
python  -c "import abel.tests; abel.tests.run_cli(coverage=True)"
```
to run tests locally.

**Comment by stggh** [comment]
Weird. I did run `./nosetests`, but now both unit testing methods generate errors. I don't think the failures are related to the `symmetry.py` changes. I will investigate.

Note also 
>Shouldn't these be included in the docs?

are also unrelated. I did a rebase before this PR.

**Comment by DanHickstein** [comment]
I think that the changes to the get_image_quadrants broke `abel.benchmark.is_symmetric`.

**Comment by stggh** [comment]
Ahh, because I changed the default `reoreint=True`

**Comment by DanHickstein** [comment]
...and that default is fine in my opinion.

**Comment by stggh** [comment]
Thanks @DanHickstein, explains why the unit tests were working for my original PR

**Comment by DanHickstein** [comment]
Merging this! Thanks @stggh!

**Comment by stggh** [comment]
@DanHickstein, super efficient.  In a new branch `no-fiabel` I have removed all the  f/iabel_  references, and made all the transforms use `<method>.<method>_transform()` as the fundamental method. I am testing before submitting a new PR. I will also look at changing `center.find_center_by_slice()` to return only the coordinates of the center.

**Comment by DanHickstein** [comment]
Sounds great @stggh!

And yes, it's efficiency time! I am very excited to get a new version released so that I can tell my co-workers to start using PyAbel.

**Pull request merged**